packages/ui/src/components/SearchPickerItem.svelte

```diff
@@ -0,0 +1,66 @@
+<!--
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { IconClose } from '..'
+</script>
+
+<button
```

Comment:
In the design layouts, the hover effect is applied to the whole badge, not to the cross.
I think that in this case it is better to use a button. It will also make it more accessible and convenient.